cirq/contrib/quantum_volume/quantum_volume.py

```diff
@@ -95,7 +95,7 @@ def compute_heavy_set(circuit: cirq.Circuit) -> List[int]:
 def sample_heavy_set(circuit: cirq.Circuit,
                      heavy_set: List[int],
                      *,
-                     repetitions=10000,
+                     repetitions=10_000,
```

Comment:
I like it.

Reply:
hashtag feature I floated in python-dev irc room a couple years ago and everyone shouted at me for saying it's dumb before implementing it a couple releases later